tests/contract/test_comms_management_send_batch_message.py

```diff
@@ -0,0 +1,62 @@
+from pact import Consumer, Provider, Term
+from pact.matchers import get_generated_values
+import pytest
+from communication_management import CommunicationManagement
+from recipient import Recipient
+
+@pytest.fixture
+def send_batch_pact():
+  pact = Consumer("MessageStatusHandler").has_pact_with(
+      Provider("CommunicationManagement"),
+      pact_dir="tests/contract/pacts",
+  )
+  pact.start_service()
+  yield pact
+  pact.stop_service()
+
+def test_send_batch_message(send_batch_pact, monkeypatch):
+    comms_management = CommunicationManagement()
+    comms_management.base_url = send_batch_pact.uri
+    comms_management.api_key = "test_api_key"
+
+    expected_reponse = {
+                "data": {
+                    "type": "MessageBatch",
+                    "attributes": {
+                        "routingPlanId": Term(r"[0-9a-zA-Z\-]{27}", "2HL3qFTEFM0qMY8xjRbt1LIKCzM"),
```

Comment:
This doesn't reflect the NHS Notify response here https://digital.nhs.uk/developer/api-catalogue/nhs-notify#post-/v1/message-batches note that routing plan is a JSON object with it's own attributes.
I think the closer to the truth we are the better with contract tests.